lib/modules/datasource/hex/schema.ts

```diff
@@ -8,9 +8,21 @@ export const HexRelease = z
     html_url: z.string().optional(),
     meta: z
       .object({
-        links: z.object({
-          Github: z.string(),
-        }),
+        links: z
+          .record(z.unknown())
```

Comment:
why use unknown here? is there anything else besides strings allowed? 🤔

Reply:
you are right, values are expected to be strings [ref](https://github.com/hexpm/hexpm/blob/47893d95cfe6024c85bf08007d6b6a41ee6bbfe1/lib/hexpm/repository/package_metadata.ex#L9), i will make the change

Reply:
Because it's irrelevant at the first stage of the validation and internally I think it'll skip unnecessary type checks for values for all keys we don't care about anyways

Reply:
ok. Anyway i assume that the type I added does not affect at all and probably gives more clarity to the future reader?

Reply:
Yeah, I don't mind this change